apps/web/ui/modals/manage-usage-modal.tsx

```
@@ -0,0 +1,277 @@
+                  value: suggestedPlan.limits.clicks,
+                  label: `total tracked events/mo`,
+                  difference: suggestedPlan.limits.clicks - (usageLimit ?? 0),
+                },
+                {
+                  icon: Hyperlink,
+                  value: suggestedPlan.limits.links,
+                  label: `new links/mo`,
+                  difference: suggestedPlan.limits.links - (linksLimit ?? 0),
+                },
+              ].map(({ icon: Icon, value, label, difference }) => (
+                <div className="text-content-default flex items-center gap-2 text-sm">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)